test.md

```diff
@@ -0,0 +1,15 @@
+just a test file
```

Comment:
I assume this is going to be removed?

Reply:
That was an old file I used to initially used for testing actions. Now deleted from this branch (and master as well).

Reply:
Re: `.DS_store`: I'm pretty sure `.gitignore` in this branch contains the correct paths ... however, the problem may have arisen because I created this branch from sih/master and then merged in changes from josephguillaume/master.